src/Repository/index.js

```diff
@@ -82,6 +82,15 @@ class Repository {
         })
     }
 
+    async getDefaultBranch() {
+        const result = await this.github.repos.get({
```

Comment:
I'm wary of introducing more network requests. It looks like we can get this from the initial payload in: https://github.com/all-contributors/all-contributors-bot/blob/master/test/fixtures/issue_comment.created.json#L181

Reply:
(we want to keep the number of API requests down, I feel like it's already pretty high)